src/e2e/app/cli/repo-tests/src/test_flow.rs

```diff
@@ -7,13 +7,23 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
-use kamu_cli_e2e_common::{AccessToken, KamuApiServerClient, KamuApiServerClientExt};
+use chrono::{TimeZone, Utc};
+use kamu_cli_e2e_common::{
+    AccessToken,
+    KamuApiServerClient,
+    KamuApiServerClientExt,
+    RequestBody,
+    DATASET_ROOT_PLAYER_SCORES_INGEST_DATA_NDJSON_CHUNK_1,
+    DATASET_ROOT_PLAYER_SCORES_INGEST_DATA_NDJSON_CHUNK_2,
+    DATASET_ROOT_PLAYER_SCORES_INGEST_DATA_NDJSON_CHUNK_3,
+};
+use opendatafabric::{DatasetAlias, DatasetName};
 use tokio_retry::strategy::FixedInterval;
 use tokio_retry::Retry;
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-pub async fn test_get_dataset_list_flows(kamu_api_server_client: KamuApiServerClient) {
+pub async fn test_web_ui_get_dataset_list_flows(kamu_api_server_client: KamuApiServerClient) {
```

Comment:
What is the motivation to rename it? There is no web ui behind it, it's just GraphQL API. Could be named as `test_gql_get_dataset_list_flows`, if you'd like to emphasize the transport mechanism.

Reply:
Since these are queries copied from `kamu-web-ui`. With the renaming, I wanted to separate the more complex tests from these.

Reply:
makes sense -- will rename the tests

Reply:
Done